web/src/components/UserProfile/UserProfile.js

```diff
@@ -48,11 +49,11 @@ export default (props) => {
 
   const userInfoButton = () => {
     if (props.me) {
-      // Save this part for later
-      // <div className="user-profile-info-button">
-      //   Edit profile
-      // </div>
-      return null
+      return (
+        <div className="user-profile-info-button" onClick={() => window.location.href = `/settings`}>
```

Comment:
nit: perhaps use Router.push so that browser history is preserved after navigating to the new page